applications/StructuralMechanicsApplication/StructuralMechanicsApplication.py

```diff
@@ -4,7 +4,7 @@
 application_folder = "StructuralMechanicsApplication"
 
 # The following lines are common for all applications
-from . import application_importer
+from .. import application_importer
```

Comment:
We cannot change this...

Reply:
#3217

Reply:
well ... that needs to be backward compatible no?

Reply:
It is, see the discussion in #3217 
Btw #3217 seems to work fine and is waiting for a closer look of the committee :)